data_steward/analytics/cdr_ops/raw_rdr_export_qc.py

```diff
@@ -1293,8 +1289,20 @@
     value_source_concept_id NOT IN (2100000008, 2100000009, 903096) -- wear_no, wear_yes, pmi_skip --
     )
 
-""").render(project_id=project_id,
-            new_rdr=new_rdr)
+""").render(project_id=project_id, new_rdr=new_rdr)
 execute(client, query)
 
+# # Check to catch duplicate observation_ids
 
+tpl = JINJA_ENV.from_string('''
+SELECT
+  observation_id,
+  COUNT(DISTINCT person_id) AS n
```

Comment:
Regardless of person_id, we should not see any duplicates in observation_id
```suggestion
  COUNT(DISTINCT observation_id) AS n
```